mlip_arena/models/externals/nequip.py

```
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from nequip.ase import NequIPCalculator
+from ase.data import chemical_symbols
+from mlip_arena.models.utils import get_freer_device
+
+
+class NequIP_OAM_L(NequIPCalculator):
+    nequip_net_id = "nequip.net:mir-group/NequIP-OAM-L:0.1"
+
+    def __new__(cls, device: str | None = None, **kwargs):
```

Ruff (0.14.0)

Unused static method argument: `kwargs`

(ARG004)